test/zdtm/transition/fifo_loop.c

```diff
@@ -125,7 +132,7 @@ int main(int argc, char **argv)
 		uint8_t rbuf[sizeof(buf)], *p;
 
 		datagen(buf, sizeof(buf), NULL);
-		wlen = write(writefd, buf, sizeof(buf));
+		wlen = write(writefd, buf, pipe_size);
```

Comment:
Maybe I'm wrong, but it looks like `min(sizeof(buf), pipe_size)` is safer.

Also, here
https://github.com/checkpoint-restore/criu/blob/0fd9f79ad03251bedc9ff653fcb94a2cbdaae15d/test/zdtm/transition/fifo_loop.c#L88

we have write with `sizeof(buf)` size. Shouldn't we need to use `min(sizeof(buf), pipe_size)` here too?